news/14043-binary-replace-newline-fix

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* Fix binary_replace not matching chunks that end with \n
```

Comment:
```suggestion
* Fix binary_replace not matching chunks that end with `\n` (#14043)
```

Just adding a link to the original issue. These will appear later in our release notes.

Reply:
Done!